<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>391</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Czesc 1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>czesc witaj 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>strona 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>strona 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
